### ISSUE TITLE
Hangs in python3 with --demangle

### ISSUE THREAD

**Issue by typelogic** [eriwen/lcov-to-cobertura-xml#34]
Works in python2.

I tried in python3 it hangs and no visible error messages. So I removed the `--demangle` argument and it does not hang and it produced the `xml`. 

Thanks men.

**Comment by typelogic** [comment]
python3 needs `sudo apt install python3-distutils`

**Comment by ensc** [comment]
a

```diff
         newname = name + "\n"
         self.pipe.stdin.write(newname.encode('utf-8'))
+        self.pipe.stdin.flush()
         res = self.pipe.stdout.readline()
         return str(res.rstrip())
```

fixes it here

**Comment by v-lopez** [comment]
Also works with python2 in case anyone is wondering, even if it's not needed.

**Comment by j4m3s** [comment]
> python3 needs `sudo apt install python3-distutils`

Unfortunately distutils isn't available on recent Ubuntu.

**Comment by eriwen** [comment]
This was fixed with ea9b81367a0c1d2551d6542b5054537eab726f8a I believe.
